build/build.go

```diff
@@ -0,0 +1,16 @@
+package build
+
+import (
+	"github.com/ipfs/go-log"
+)
+
+var (
+	Version  = "unknown" // Client software version. Set with `-ldflags -X` during the build.
+	Revision = "unknown" // Client software revision. Set with `-ldflags -X` during the build.
+	logger   = log.Logger("keep-build")
+)
+
+func LogVersion() {
+	logger.Info("version=", Version)
+	logger.Info("revision=", Revision)
+}
```

Comment:
I don't think we should log the version via logger on every binary execution. Version is available with `--version` command and the fact we log version and revision via logger when executing every possible command does not look good IMO:
```
❯ ./keep-client --version
2022-09-11T16:36:05.007+0200    INFO    keep-build      build/build.go:14       version=v1.3.1-5904-gafea16360
2022-09-11T16:36:05.008+0200    INFO    keep-build      build/build.go:15       revision=afea16360
keep-client version v1.3.1-5904-gafea16360 (revision afea16360)
```

```
❯ ./keep-client --help
2022-09-11T16:34:57.331+0200    INFO    keep-build      build/build.go:14       version=v1.3.1-5904-gafea16360
2022-09-11T16:34:57.331+0200    INFO    keep-build      build/build.go:15       revision=afea16360
Command line interface (CLI) for running a Keep provider

Usage:
  keep-client [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  ethereum    Provides access to Keep network Ethereum contracts.
  help        Help about any command
  ping        bidirectional send between two peers to test the network
  start       Starts the Keep Client

Flags:
  -c, --config string   Path to the configuration file. Supported formats: TOML, YAML, JSON.
      --developer       Developer network
      --goerli          Görli network
  -h, --help            help for keep-client
      --mainnet         Mainnet network
  -v, --version         version for keep-client

Use "keep-client [command] --help" for more information about a command.
```

tl;dr; I'd remove `LogVersion` and the call to it from `main.go`.

Reply:
Sure [7ab8b8d](https://github.com/keep-network/keep-core/pull/3234/commits/7ab8b8d1e59184c7c906acebde0f810c66967a6c)